src/main/java/org/troyargonauts/subsystems/DriveTrain.java

```diff
@@ -13,19 +13,20 @@
 import org.troyargonauts.Robot;
 
 /**
- * using PID and stating our speed, turn, and nerf we made a code to run our 8 wheel tank drivetrain with 2 motors
+ * Drivetrain uses PID as well as runs with certian speed and turn to reach its desired target
```

Comment:
"Drivetrain allows control of the robot's drivetrain in an arcade drive (cheesy drive) and tank drive. Includes a distance PID and a turn PID using the Pigeon 2 from CTRE" Something like this. Describe the entire class in 2-3 sentences, not just a feature.